### ISSUE TITLE
'Yarn liquidator' won't start any longer ... gives 'Error: [big.js] Invalid number

### ISSUE THREAD

**Issue by perezsan** [port-finance/liquidator#25]
I get this error when running the latest git pull; nothing else has been changed or updated on my system:

```
yarn run v1.22.17
$ ts-node src/partial.ts
Port liquidator launched on cluster=https://solana-api.projectserum.com
unknown error: Error: [big.js] Invalid number
Error: [big.js] Invalid number
    at parse (/home/lq/liquidator/node_modules/big.js/big.js:142:13)
    at new Big (/home/lq/liquidator/node_modules/big.js/big.js:109:9)
    at Big.P.div (/home/lq/liquidator/node_modules/big.js/big.js:346:16)
    at generateEnrichedObligation (/home/lq/liquidator/src/partial.ts:361:8)
    at /home/lq/liquidator/src/partial.ts:290:7
    at Array.map (<anonymous>)
    at getUnhealthyObligations (/home/lq/liquidator/src/partial.ts:289:6)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at runPartialLiquidator (/home/lq/liquidator/src/partial.ts:115:36)
```

**Comment by perezsan** [comment]
note: I've also tried the liquidator on a fresh install of alpine 3.14.2, but only under docker, and still get the same [big.js] error.

**Comment by leonardgee** [comment]
Should be fixed by #26

**Comment by perezsan** [comment]
'Tis! Thank you.
